manifests/h/HouseOfLife/BitLord/2.4.6.356/HouseOfLife.BitLord.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HouseOfLife.BitLord
+PackageVersion: 2.4.6.356

```

```suggestion
PackageVersion: 2.4.6-356
```
to match Control Panel.